index.html

```diff
@@ -48,112 +54,156 @@ <h2>I’am a Software Developer</h2>
+    <div class="works">
+        <div class="recent">
+            <div class="title">
+                <h2 class="first">My Recent Works</h2>
+            </div>            
+            <hr class="first-line">
+        </div>
+            
+        <div class="cards">
+           <div class="one">
+                    <div class="this-pic">
+                        <img class="picture" src="image/ImgPlaceholder.png" alt="icon">
+                   </div>
+             <div class="post">
+                <h3 class="multi">Multi-Post Stories</h3>
+                <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                </p>
+                <div class="languages">
+                    <ul>
+                        <li class="css">css<a href="#"></a></li>
+                        <li class="html">html<a href="#"></a></li>
+                        <li class="bootstrap">bootstrap<a href="#"></a></li>
+                        <li class="ruby">ruby<a href="#"></a></li>
+                        </ul>
+                </div>
+                <div><button  type="button" class="first-btn">See Project</button></div>
+              </div>
+            </div>
+             <div class="grid-container">
+                <div class="card1">
+                    <div class="style-grid">
+                            <div class="profession profession1">
+                                <div class="first-child">
+                                    <h4 class="pro">Profesional Art Printing Data</h4>
+                                <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                                </p>
+                                <ul>
+                                    <li class="html2">html</li>
+                                    <li class="bootstrap2">bootstrap</li>
+                                    <li class="Ruby2">ruby</li>
+                                </ul>
+                                </div>        
+                            </div>
+                        <button class="second-btn" type="button">See Project</button>
+                    </div>
+                    <div class="style-grid">
+                        <div class="profession profession2">
+                            <h4 class="pro">Profesional Art Printing Data</h4>
+                            <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                            </p>
+                            <ul>
+                                <li class="html2">html</li>
+                                <li class="bootstrap2">bootstrap</li>
+                                <li class="Ruby2">ruby</li>
```

Comment:
- [ ] Please note that the `See Project` show up only if the user hovers on the card, so basically all the cards should be like the default style below and only when the user hover the dark background get removed and the button appears

Default style            |  On hover
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/188262872-e150a881-8fd3-43d0-86a0-1a36f965e428.png) |  ![image](https://user-images.githubusercontent.com/89282221/188262884-51e0f6e4-a614-43ef-ac6a-719925499bdd.png)